setup.py

```diff
@@ -22,6 +25,8 @@
     license='BSD+',
     packages=['gnlpy'],
     package_dir={'gnlpy': '.'},
+    install_requires=requirements.txt,
+    include_package_data=True,
```

Comment:
without the manifest file, this won't be needed.

Reply:
after introducing MANIFEST.in it becomes necessary to add `include_package_data=True`